kobo/settings/base.py

```diff
@@ -438,7 +464,7 @@
         {'widget': 'django.forms.Textarea'},
     ],
     'mfa_help_text_fields_jsonschema': [
-        'kpi.fields.jsonschema_form_field.MfaHelpTextField',
+        'kpi.fields.jsonschema_form_field.CustomTextWithTranslations',
         {'widget': 'django.forms.Textarea'},
     ],
```

Comment:
Good idea to rename `MfaHelpTextField'`. 👏 ❤️ 

You can get rid of this block and use on the one you added for `custom_password_guidance_text_field`. 
But please rename `custom_password_guidance_text_field` to `custom_text_with_translations`. Then use it for both MFA and Custom password guidance.